entrypoint.sh

```diff
@@ -1,33 +1,36 @@
 #!/bin/sh
 
 set -e
-
-if [ ! -z "$INPUT_FRAMEWORK" ] && [ ! -z "$INPUT_CONTROL" ]; then
-echo "Framework and Control is specified. Please specify either one of them or neither"
-exit 1
-fi
-
-# Split the controls by comma and concatenate with quotes around each control
-if [ ! -z "$INPUT_CONTROL" ]; then
-    CONTROLS=""
-    set -f; IFS=','
-    set -- $INPUT_CONTROL
-    set +f; unset IFS
-    for control in "$@"
-    do
-        control=$(echo $control | xargs) # Remove leading/trailing whitespaces
-        CONTROLS="$CONTROLS\"$control\","
-    done
-    CONTROLS=$(echo "${CONTROLS%?}")
-fi
-
-FRAMEWORK_CMD=$([ ! -z "$INPUT_FRAMEWORK" ] && echo "framework $INPUT_FRAMEWORK" || echo "")
-CONTROL_CMD=$([ ! -z "$INPUT_CONTROL" ] && echo control $CONTROLS || echo "")
-
-COMMAND="kubescape scan $FRAMEWORK_CMD $CONTROL_CMD $INPUT_FILES --fail-threshold $INPUT_THRESHOLD $INPUT_ARGS"
-
-if [ ! -z "$INPUT_EXCEPTIONS" ]; then
-COMMAND="${COMMAND} --exceptions ${INPUT_EXCEPTIONS}"
+if [ "$INPUT_SCANREPOSITORY"  = "no" ]; then 
+  if [ ! -z "$INPUT_FRAMEWORK" ] && [ ! -z "$INPUT_CONTROL" ]; then
+  echo "Framework and Control is specified. Please specify either one of them or neither"
+  exit 1
+  fi
+
+  # Split the controls by comma and concatenate with quotes around each control
+  if [ ! -z "$INPUT_CONTROL" ]; then
+      CONTROLS=""
+      set -f; IFS=','
+      set -- $INPUT_CONTROL
+      set +f; unset IFS
+      for control in "$@"
+      do
+          control=$(echo $control | xargs) # Remove leading/trailing whitespaces
+          CONTROLS="$CONTROLS\"$control\","
+      done
+      CONTROLS=$(echo "${CONTROLS%?}")
+  fi
+
+  FRAMEWORK_CMD=$([ ! -z "$INPUT_FRAMEWORK" ] && echo "framework $INPUT_FRAMEWORK" || echo "")
+  CONTROL_CMD=$([ ! -z "$INPUT_CONTROL" ] && echo control $CONTROLS || echo "")
+
```

Comment:
Since all the arguments and options can be passed for a repository scan. You can add the if condition here and provide either `$INPUT_FILES` or `https://github.com/$GITHUB_REPOSITORY` and `$INPUT_ACCOUNTGUID` accordingly. Ensure that the `INPUT_FILES` is present if `INPUT_SCANREPOSITORY` is set to no and `INPUT_ACCOUNTGUID` is present if set to yes.

Reply:
What should happen when say scanRepo is set to `yes` and accountID is not provided?

Reply:
There should be an error like https://github.com/kubescape/github-action/blob/90d242dccc609649216f45856e5d230a4a153282/entrypoint.sh#L6

Reply:
Updated the file. Could you please review the changes, So that I can solve the merge conflict.